tests/tacacs/test_accounting.py

```diff
@@ -34,6 +35,10 @@ def check_tacacs_server_log_exist(ptfhost, tacacs_creds, command):
             Find logs match following format: "tacacs_rw_user ... cmd=command"
             Print matched logs with /P command.
     """
+    
+    # On some hardware, syslog write to log file very slow, wait 3 seconds untill log file ready.
+    time.sleep(3)
```

Comment:
>3 [](http://example.com/codeflow?start=15&length=1)

Fixed sleep is not recommended. In slow hardware, you tend to set longer value to cover the worst case. In fast hardware, you tend to set shorter value to be efficient.

There are similar functions like `wait_for_***`, you can set a relative large max value, and it will return as soon as possible when the event occur.

Reply:
Fixed, add new method because not find a existing suitable method.